Makefile

```diff
@@ -22,9 +23,15 @@ clean-pyc:
 	find . -name '*.pyo' -exec rm -f {} +
 	find . -name '*~' -exec rm -f {} +
 
+format: 
+	python3 -m black probe_scraper tests setup.py
+	python3 -m isort probe_scraper tests setup.py
+
 lint: build
 	docker-compose run app flake8 --max-line-length 100 .
 	docker-compose run app yamllint repositories.yaml .circleci
+	docker-compose run app python -m black --check probe_scraper tests setup.py
```

Comment:
One last file to check is the `conftest.py` at the root of the project (which should be moved to tests/, but that's unrelated to this PR).